spring-integration-core/src/main/java/org/springframework/integration/support/StringObjectMapBuilder.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.support;
+
+/**
+ * A map builder creating a map with String keys and values.
+ *
+ * @author Gary Russell
+ *
+ * @since 4.3.17
+ */
+public class StringObjectMapBuilder extends MapBuilder<StringObjectMapBuilder, String, Object> {
```

Comment:
Yeah... We can't do this for the `4.3.x` - the `MapBuilder` has appeared here since `5.0`.

Reply:
np; the tests don't exist in 4.3 anyway.